src/main/pg/stats.c

```diff
@@ -34,5 +34,6 @@ PG_RESET_TEMPLATE(statsConfig_t, statsConfig,
     .stats_total_flights = 0,
     .stats_total_time_s = 0,
     .stats_total_dist_m = 0,
+    .stats_total_packs = 0,
```

Comment:
PG version should probably updated on line 30:
`PG_REGISTER_WITH_RESET_TEMPLATE(statsConfig_t, statsConfig, PG_STATS_CONFIG, 3);`

Reply:
Done in [68549ae](https://github.com/betaflight/betaflight/pull/10808/commits)/68549ae7fd8c06b3ca56093a8ffee0d155c27340).

I'll really read more about PG version, when to change and when not. I think I will have the same issue in another PR.